packages/frontend/src/util/Api.js

```diff
@@ -66,8 +70,8 @@ const getDataContractByIdentifier = (identifier) => {
     return call(`dataContract/${identifier}`, 'GET')
 }
 
-const getDataContracts = (page = 1, limit = 30, order = 'asc') => {
-    return call(`dataContracts?page=${page}&limit=${limit}&order=${order}`, 'GET')
+const getDataContracts = (page = 1, limit = 30, order = 'asc', orderBy = '') => {
```

Comment:
I don't think empty string does anything here, let's just leave it as `orderBy` (will appear as undefined in case it is not passed)